evm.md

```diff
@@ -1779,6 +1801,7 @@ Overall Gas
     rule <k> #gas [ OP , AOP ]
           => #if #usesMemory(OP) #then #memory [ AOP ] #else .K #fi
           ~> #gas [ AOP ]
+          ~> #if #usesAccessList(OP) #then #access [ AOP ] #else .K #fi
```

Comment:
Any reason to do access list account after execution gas? I guess I would put it even before the memory accounting.

Reply:
```suggestion
    rule <k> #gas [ OP , AOP ]
          => #if #usesMemory(OP) #then #memory [ AOP ] #else .K #fi
          ~> #gas [ AOP ]
          ~> #if Ghasaccesslist << SCHED >> andBool #usesAccessList(OP) #then #access [ AOP ] #else .K #fi
```

Reply:
Here is where you can inline the condition which checks whether we're on an access list schedule.

Reply:
> Any reason to do access list account after execution gas? I guess I would put it even before the memory accounting. 

Having it executed before `#gas` would mess the gas calculation for `CALL*` opcodes, which take in consideration the `touchedAccounts` Set.  In the case of a cold account, `#access` would mark it as warm before the gas calculation in `#gas`. 

> Here is where you can inline the condition which checks whether we're on an access list schedule.

Applied.